java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/ArrayConsumer.java

```diff
@@ -28,7 +28,7 @@
  * Consumer which consume array type values from {@link ResultSet}.
  * Write the data to {@link org.apache.arrow.vector.complex.ListVector}.
  */
-public abstract class ArrayConsumer implements JdbcConsumer<ListVector> {
+public abstract class ArrayConsumer extends  BaseConsumer<ListVector> {
```

Comment:
spacing looks off

Reply:
There is an extra space here. Thanks for the careful review. Maybe we need to enhace our style checking.

Reply:
Yes, https://github.com/revelc/formatter-maven-plugin might be an option for this but we should discuss on the mailing list first (I would hope that we can configure it so not too many files need to change on the first pass).

Reply:
Thanks for the suggested plugin. I agree with you that we should enhance the style checking in small steps.